recipes/sqlite3mc/all/conandata.yml

```diff
@@ -1,4 +1,16 @@
 sources:
+  "1.8.4":
+    url: "https://github.com/utelle/SQLite3MultipleCiphers/archive/refs/tags/v1.8.4.tar.gz"
+    sha256: "453e1938a02c91796a013169d56f746f153d7c7a77b59894bf8462b341a343ca"
+  "1.8.3":
+    url: "https://github.com/utelle/SQLite3MultipleCiphers/archive/refs/tags/v1.8.3.tar.gz"
+    sha256: "1a9c26082068ee36c33b0a4e061d013db345ad170bab0ab00c8dda2c7c96561a"
+  "1.8.2":
+    url: "https://github.com/utelle/SQLite3MultipleCiphers/archive/refs/tags/v1.8.2.tar.gz"
+    sha256: "c349f45505641a3ee3e5faa4498a18317b65697e9a3cbfbe41a6612e44389c55"
+  "1.8.1":
+    url: "https://github.com/utelle/SQLite3MultipleCiphers/archive/refs/tags/v1.8.1.tar.gz"
+    sha256: "44215d5812ec2cfa1a842d9aae2908be0b14dce4231bc3243394b24b3affb78a"
```

Comment:
```suggestion
```
Intermediate versions are usually only added when there's a specific need for them, especially in the case of patch-version bumps. Please skip them.

Reply:
> Intermediate versions are usually only added when there's a specific need for them, especially in the case of patch-version bumps. Please skip them.

Please explain what exactly is a _specific need_? Who needs what? And who decides what is needed and what not?

And please explain why you think the intermediate versions are just patch-version bumps?

Typically, a new version of _SQLite3 Multiple Ciphers_ is released shortly after a new version of _SQLite_ was released. And that is what most users of _SQLite3 Multiple Ciphers_ expect: encryption support for the latest SQLite version.

| SQLite3mc version | based on SQLite3 version |
| --- | --- |
| 1.8.4 | 3.45.2 |
| 1.8.3 | 3.45.1 |
| 1.8.2 | 3.45.0 |
| 1.8.1 | 3.44.2 |
| 1.8.0 | 3.44.1 |

The above table lists for each version the related _SQLite3_ version. Looking at the Conan recipe for _SQLite3_, I see that **all** versions mentioned in the table are included - without exception.

So, IMHO it would be somewhat logical to support them all. However, if you insist I can remove the "intermediate" versions.

And what should be done in the future? Should this recipe be updated after a release of a new SQLite version or not? Probably not, because according to your argument those updates would be only patch-version bumps.

Reply:
We actually just avoid pushing multiple patches due lack of usage, like people will prefer download the latest version (based on download numbers in Conan Center). So publishing multiple versions will only result in more packages in the repository (Sqlite3mc will produce around 100 binaries packages for each version), but most probably people will prefer the latest. Of course, if there is a reason of keep all versions, like someone need that version for a project, we will not refuse keeping it.

As SQlite3 is used not only largely in ConanCenterIndex, but also uses multiple versions, I don't see all problem keeping all versions for this PR, but because is a project strict related to another project tags.